data_steward/bq_utils.py

```diff
@@ -41,6 +41,24 @@ def get_table_id(hpo_id, table_name):
     return hpo_id + '_' + table_name
 
 
+def get_table_info(table_id, dataset_id=None, project_id=None):
+    """
+    Get metadata describing a table
+
+    :param table_id: ID of the table
+    :param dataset_id: ID of the dataset containing the table (EHR dataset by default)
+    :param project_id: associated project ID (default app ID by default)
+    :return:
+    """
+    bq_service = create_service()
+    if project_id is None:
+        project_id = app_identity.get_application_id()
+    if dataset_id is None:
+        dataset_id = get_dataset_id()
+    job = bq_service.tables().get(projectId=project_id, datasetId=dataset_id, tableId=table_id)
+    return job.execute(num_retries=BQ_DEFAULT_RETRY_COUNT)
+
+
 def load_csv(schema_path, gcs_object_path, project_id, dataset_id, table_id, write_disposition='WRITE_TRUNCATE',
```

Comment:
Is this used to create new tables, or just load CSV into existing ones?

(If it's the latter, you don't need to bother with the clustering stuff... clustering only happens at creation time.)

Reply:
Yes, I think you're right. I will confirm and fix. Thanks @danqrodney! Btw, you may already know this, but TIL from the [circle build](https://circleci.com/gh/all-of-us/curation/59?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link#action-21) that querying tables with clustering is not supported in Legacy SQL. I do not believe we need Legacy SQL in this project (will check).

Reply:
Yeah, we don't use legacy SQL for anything. It's probably better to avoid it entirely.